src/assets/asset-transfers.md

```diff
@@ -97,6 +99,13 @@ The above example has $10 left on a $60 dollar giftcard at the time of transfer.
 | 403    | {% break _ INSUFFICIENT_WALLET_BALANCE %} | The value of the asset-transfer exceeds the balance on the wallet |
 | 403    | {% break _ QUOTA_EXCEEDED %} | The transfer exceeds one or more spend quota limits see [ Quota Error Response ]({% link quotas.md %}#quota-error-response)  |
 
+**Example webhook response**
+
+TBD
+```json
+{}
+```
```

Comment:
I think you should have a chat with @ncjones on webhooks, he's raised an issue with the way we're doing webhooks.

Reply:
This may affect when we do this-- not saying no, just saying chat :)

Reply:
Discussed this, we're doing it.